packages/code-du-travail-api/src/server/routes/search/search.getSavedResult.js

```diff
@@ -11,11 +11,9 @@ const getSavedResult = (query, excludeSources = []) => {
     // apply source filters if any
     // build facets
     return {
-      hits: {
-        hits: knownQuery.refs.filter(
-          ref => !excludeSources.includes(ref._source.source)
-        )
-      },
+      items: knownQuery.refs.filter(
+        ref => !excludeSources.includes(ref._source.source)
```

Comment:
I wonder if we should remove also exludesSource from the AIP or no (I think it could be useful but probably later)

Reply:
It is used currently in the theme page to fetch search result by excluding themes !

Reply:
will disappear with the new themes data